db/schema.rb

```diff
@@ -411,6 +411,7 @@
     t.datetime "call_start_time"
     t.json "accepted_offer_details"
     t.boolean "pay_in_installments", default: false, null: false
+    t.integer "fixed_duration_months"
```

Comment:
According to your migration, the schema should have an index declaration for this table and the `fixed_duration_months`. Curious as to why it's not there, but as stated above, you should remove that index from the migration file anyway since it is not used.